rasa/cli/data.py

```diff
@@ -226,12 +229,21 @@ def _validate_story_structure(validator: "Validator", args: argparse.Namespace)
 
 def _convert_nlu_data(args: argparse.Namespace) -> None:
     import rasa.nlu.convert
+    from rasa.nlu.training_data.converters.watson_nlu_json_to_yaml_converter import (
+        WatsonTrainingDataConverter,
+    )
 
     if args.format == "json":
         rasa.nlu.convert.convert_training_data(
             args.data, args.out, args.format, args.language
         )
         telemetry.track_data_convert(args.format, "nlu")
+    elif args.format == "yaml":
+        is_watson = WatsonTrainingDataConverter().filter(args.data)
```

Comment:
?
```suggestion
        is_watson = WatsonTrainingDataConverter().filter(Path(args.data))
```

Reply:
forgot about this one. just committing again